File: fizzbuzz.py
'''
Program outputs the first 100 fizzbuzz numbers

Author: Kolawole Andrew
github: github.com/techkaduna

'''

# declare counter variable
i = 1

# using while loop to check condition continously
while (i <= 100):
    #checking if i is divisible by 3 and printing 'fizz'
    if (i % 3) == 0: # checking if i is perfectly divisible by 3
        print("Fizz")
    elif (i % 5) == 0: # checking if i is perfectly divisible by 5
        print("Buzz")
    else: # i is not divisible by 3 nor 5
        print(i)

    # increasing the counter
    i = i + 1

# Note: We could have used python's in-built range() method

File: fizzbuzz.java
/**
 * Program prints first 100 FizzBuzz numbers
 *
 * Author: Kolawole Andrew
 */


class fizzbuzz {
    public static void main(String[] args) {
        // initializing and declaring counter variable
        int i = 1;
        
        // checking condition continously
        while (i <= 100){
            // check if i is perfectly divisible by 3
            if (i % 3 == 0){
                System.out.println("Fizz");
            } else if (i % 5 == 0){
                // check if i is perfectly divisible by 5
                System.out.println("Buzz");
            } else{
                // check if i is not perfectly divisible by 3 or 5
                System.out.println(i);
            }
            // increment counter variable
            i++;
        }
    }
}
